Templates/ScriptCanvasNode/template.json

```diff
@@ -0,0 +1,50 @@
+{
+    "template_name": "ScriptCanvasNode",
+    "origin": "Open 3D Engine - o3de.org",
+    "origin_url": "https://github.com/o3de/o3de",
+    "license": "Apache-2.0 or MIT",
+    "license_url": "https://github.com/o3de/o3de/blob/development/LICENSE.txt",
+    "display_name": "Default Component Template",
+    "summary": "A template for a minimal Script Canvas node",
+    "canonical_tags": [
+        "Template"
+    ],
+    "user_tags": [
+        "ScriptCanvasNode"
+    ],
+    "icon_path": "preview.png",
+    "copyFiles": [
+        {
+            "file": "Source/${Name}.cpp",
+            "isTemplated": true
+        },
+        {
+            "file": "Include/${Name}.h",
+            "isTemplated": true
+        },
+        {
+            "file": "Source/${Name}.ScriptCanvasNodeable.xml",
+            "isTemplated": true
+        },
+        {
+            "file": "${Name}_files.cmake",
+            "isTemplated": true
+        },
+        {
+            "file": "CMakeLists.txt",
+            "isTemplated": true
+        },
+        {
+            "file": "README.md",
+            "isTemplated": true
```

Comment:
It's a bit odd that the `README.md` is a templated file.  Its purpose is to instruct users how to produce an instance of this template.  In order to produce a key-value-substituted copy of this `README.md`, wouldn't users already have ran `create-from-template`?  A bit of a chicken-egg scenario.  I'd suggest making `README.md` a generic instruction reference and make it non-templated.

Reply:
It serves two purposes, you can read it from the Templates folder and it has instructions on "Before you run the template" then it generates a new README.md in the target project (vcproj) that contains the exact command lines you can copy/paste to register the new module. I could split it into 2 readmes, it just seemed confusing as well.

Reply:
No, don't need to split it up into two files.  Your explanation is fine, I was just noting that in the `create-from-template` command example, it has a templated parameter.